academic_helper/views/courses.py

```diff
@@ -50,8 +50,13 @@ def post(self, request: WSGIRequest):
         if not request.is_ajax():
             raise NotImplementedError()
         text = request.POST["free_text"]
+        school = request.POST["school"]
+        faculty = request.POST["faculty"]
         log.info(f"Searching for {text}...")
-        queryset = Course.find_by(text)
+        log.info(f"Searching for school {school}...")
+        log.info(f"Searching for faculty {faculty}...")
```

Comment:
Unite 3 log rows to the same message: `log.info("Searching for {text}, {school}, {faculty}")`, and maybe move this row inside `find_by` function (we want this log to be written every time we search, no matter where we call the function from).